pokedex_async_redux/lib/features/pokemon_details/pokemon_details_page.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:dartx/dartx.dart';
+import 'package:pokedex_async_redux/api/model/details.dart';
+import 'package:pokedex_async_redux/features/pokemon_details/widget/pokemon_ability.dart';
+import 'package:pokedex_async_redux/features/pokemon_details/widget/pokemon_general.dart';
+import 'package:pokedex_async_redux/features/pokemon_details/widget/pokemon_moves.dart';
+import 'package:pokedex_async_redux/features/pokemon_details/widget/pokemon_stats.dart';
+import 'package:pokedex_async_redux/utilities/async.dart';
+import 'package:pokedex_async_redux/utilities/colors.dart';
+import 'package:pokedex_async_redux/utilities/constants.dart';
+import 'package:flutter_icons_null_safety/flutter_icons_null_safety.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
+
+class PokemonDetailsPage extends StatelessWidget {
+  const PokemonDetailsPage({
+    required this.pokemonDetails,
+    required this.pokeName,
+    required this.pokemonImage,
+    Key? key,
+  }) : super(key: key);
+
+  final Async<PokemonDetails> pokemonDetails;
+  final String pokeName;
```

Comment:
pass the whole pokemon object here instead and also place it above `pokemonDetails` property 🙂